cnv/cnv_users_guide/cnv-importing-virtual-machine-images-datavolumes.adoc

```diff
@@ -0,0 +1,28 @@
+[id="cnv-importing-virtual-machine-images-datavolumes"]
+= Importing virtual machine images with DataVolumes
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-importing-virtual-machine-images-datavolumes
+toc::[]
+
+You can import an existing virtual machine image into your {product-title}
+cluster. {ProductName} uses DataVolumes to automate the import of data and the
+creation of an underlying PersistentVolumeClaim.
+
+[CAUTION]
+====
+When you import a disk image into a PersistentVolumeClaim, the disk image is
+expanded to use the full storage capacity that is requested in the PVC. To use
+this space, the disk partitions and file system(s) in the virtual machine
```

Comment:
Might need to say `fully` in here. If the filesystem isn't expanded, then some of the content can still be used, correct?

Reply:
Can you be more specific about where you think `fully` should go? I'm not sure I understand.

Reply:
I believe I was mistaken here. Upon my first read I was thinking that the disk image could be utilized, just not to the full amount requested by the PVC. Rereading this section, it seems that the filesystem will need to contain the full PVC, and must be expanded if it is not large enough already.

I'm closing this comment.